src/main/java/org/helpberkeley/memberdata/ControlBlock.java

```diff
@@ -113,17 +113,13 @@ private void auditColumnNames() {
 
     public static ControlBlock create(String csvData) {
 
-        // Normalize lines
-        String normalized = csvData.replaceAll("\\r\\n?", "\n");
```

Comment:
This project produces Unix style output.  End of line is marked by a single '\n'.  Windows generally accepts this, but by default produces Windows style end of line, which is '\r' '\n'.  Throughout the project we see this replaceAll when dealing with external data.  Removing this here implies that we trust that CSVReader is doing this.  There is an old FIX THIS comment is UserExporter.auditWorkflowData() where I ask this question.  OpenCSV documentation does not seem to explicitly talk about this, but if under the covers it is using [java.io.BufferedReader.readLine()](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/BufferedReader.html#readLine()) then this is guaranteed. 

Please add CSVListReaderTest.java and some test cases verifying this behavior, and then we should be able to go through the code and remove any of the replaceAlls that are feeding CSVReader.

Reply:
I was curious about this and just wrote a set of tests (creating CSVListReaderTest.java) that show CSVReader is handling this for us. I will check that in next (along with a pair tests to improve code coverage in ControlBlock.